Tools/TestWebKitAPI/Tests/WebKitCocoa/ScriptTelemetryTests.mm

```diff
@@ -23,6 +23,6 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/ScriptTelemetryTestsAdditions.mm>)
+#if USE(APPLE_INTERNAL_SDK) && !PLATFORM(WATCHOS) && __has_include(<WebKitAdditions/ScriptTelemetryTestsAdditions.mm>)
```

Comment:
Is there a reason why we're checking the platform here, instead of inside the additions file?

(I also wonder if it may be cleaner to just turn off `ENABLE_SCRIPT_TELEMETRY` on watchOS for consistency with `ENABLE_ADVANCED_PRIVACY_PROTECTIONS`)